pyvrp/cpp/ProblemData.h

```diff
@@ -94,8 +98,9 @@ class ProblemData
         Duration const twEarly;      // Earliest possible start of service
         Duration const twLate;       // Latest possible start of service
         Duration const releaseTime;  // Earliest possible time to leave depot
-        Cost const prize = 0;        // Prize for visiting this client
-        bool const required = true;  // Must client be in solution?
+        Cost const prize;            // Prize for visiting this client
+        std::string const name;      // Location name (for reference)
```

Comment:
A `std::string` is really big, so I'm running a benchmark to check if that matters. If it does, I'll use a `char *` instead and do a little memory allocation to make that work. We don't really _need_ a `std::string`, but it's convenient and avoids managing memory manually. But I'm not willing to sacrifice any performance for this.